components/match2/wikis/clashofclans/match_group_input_custom.lua

```diff
@@ -113,7 +113,8 @@ function CustomMatchGroupInput.getResultTypeAndWinner(data, indexedScores)
 	-- Map or Match is marked as finished.
 	-- Calculate and set winner, resulttype, placements and walkover (if applicable for the outcome)
 	elseif Logic.readBool(data.finished) then
-		if MatchGroupInput.isDraw(indexedScores) then
+		local winner = tonumber(data.winner)
+		if winner == 0 or not winner and MatchGroupInput.isDraw(indexedScores) then
```

Comment:
to please the linter
```suggestion
		if tonumber(data.winner) == 0 or not Logic.isNumeric(data.winner) and MatchGroupInput.isDraw(indexedScores) then
```
and kick line above

-----
alternatively
```suggestion
		if CustomMatchGroupInput.isDraw(indexedScores, tonumber(data.winner)) then
```
and kick line above
and add
```lua
---@param indexedScores table[]
---@param winner integer?
---@return boolean
function CustomMatchGroupInput.isDraw(indexedScores, winner)
	if winner == 0 then return true end
	if winner then return false end
	return MatchGroupInput.isDraw(indexedScores)
end
```

Reply:
I see this comments was closed, but not implemented?

Reply:
hjp provided to solution methods, I tried with the first one that is ```if tonumber(data.winner) == 0 or not Logic.isNumeric(data.winner) and MatchGroupInput.isDraw(indexedScores) then``` and it managed to fixed the issue already. 

So I didnt went with the alternative method since its already fixed

Reply:
Ah I only really saw the "alternative" (missing "original") my bad. I would say the alternative is a better solution (because it's a lot easier to read and understand) compared to the first one.